src/http/http-server-base.ts

```diff
@@ -94,7 +94,8 @@ export class HttpServerBase<T extends http.Server | https.Server> {
       method: req.method,
       url: req.url,
       headers: headers,
-      body: await readBody(req)
+      body: await readBody(req),
+      receivedAt: new Date()
```

Comment:
We should probably strip this value in the `get[...]Requests` as it will otherwise mess with snapshots.

Reply:
I could also change it to a number instead?

Reply:
Yeah, that should work as well.

Reply:
Done